src/app/courses/courses-filter/courses-filter.component.ts

```diff
@@ -6,10 +6,14 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./courses-filter.component.scss']
 })
 export class CoursesFilterComponent implements OnInit {
-
+  searchText: any;
```

Comment:
Please, try to avoid usage of "any" type. In this case we use string type

Reply:
Noted. Updated